react-responsive-modal/src/index.tsx

```diff
@@ -187,7 +188,7 @@ export const Modal = ({
   useModalManager(refModal, open);
 
   // Hook used to manage the scroll
-  useScrollLock(refModal, open, showPortal, blockScroll);
+  useScrollLock(refModalContainer, open, showPortal, blockScroll);
```

Comment:
I don't really understand this change, you are passing `refModalContainer` instead of `refModal`, but on line 307 you actually changed the name, so it means you are passing the exact same value as before. I don't get how this can fix the problem?

Reply:
I see. The scroll lock somehow get fixed on my site, and probably not relating to this commit.
I'll look into it again.

I'll close the PR for now.